scripts/terraform/appservice/terraform.tfvars

```diff
@@ -1,7 +1,7 @@
-location = "canada central"
+location = "Canada Central"
 resource_group_name = "cdscracollab-innovation-rg"
 appservice_plan_id = "alphaPlan"
-appservice_name = "claim-tax-benefits-test"
```

Comment:
Do we want to remove the '-test'? I thought this was in here so we could set up other app services.

Reply:
Good catch. We can change it no problem. You are right, the goal was to actually avoid overriding what we have on "prod" now.

Reply:
Should we do that with the other names too? I mean, I don't think we need to but either works.